README.md

```diff
@@ -65,11 +65,12 @@ accept 3rd party Reporter implementations.
 
 #### HTTP Reporter
 Most common Reporter type used by Zipkin users transporting Spans to the Zipkin
-server using JSON over HTTP.
+server using JSON over HTTP. The reporter holds a buffer and reports to the backend
+asynchronously, you can se the default values [here](https://github.com/openzipkin/zipkin-go/blob/master/reporter/http/http.go#L20).
```

Comment:
I would leave out the text on default values... overly verbose for the readme...